schema.json

```diff
@@ -1725,6 +1714,9 @@
             ]
         }
     },
+    "$id": "https://citation-file-format.github.io/1.2.0/schema.json",
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
```

Comment:
I don't think there is an unambiguous sorting order based on themes or categories -- different people expect things to be in different places. Hence the alphabetical ordering safeguarded by the linter.

Reply:
Just to verify, you're ok with switching to a newer version of JSONSchema?

i.e. 

```json
"$schema": "https://json-schema.org/draft/2019-09/schema",
```

instead of 

```json
"$schema": "http://json-schema.org/draft-07/schema",
```